packages/cms-lib/personalAccessKey.js

```diff
@@ -50,15 +51,13 @@ async function getAccessToken(personalAccessKey, env = 'PROD') {
   }
 }
 
-async function refreshAccessToken(personalAccessKey, env = 'PROD') {
+async function refreshAccessToken(personalAccessKey, env = PROD) {
   const { accessToken, expiresAt, portalId } = await getAccessToken(
     personalAccessKey,
     env
   );
-  const config = getPortalConfig(portalId);
 
   updatePortalConfig({
-    ...config,
```

Comment:
It looks like after the changes to `updatePortalConfig` this call will not work as expected due to `tokenInfo` being a top-level key.